contrib/kube-prometheus/examples/kubeadm.jsonnet

```diff
@@ -0,0 +1,2 @@
+(import "kube-prometheus/kube-prometheus.libsonnet") +
```

Comment:
Why is `kubeadm.jsonnet` in `contrib/kube-prometheus/examples/` and  `kube-prometheus-minikube.jsonnet` in `contrib/kube-prometheus/hack/scripts`?

Reply:
minikube is a combination of kubeadm+node-ports, which is an opinionated way of running this stack, and nicely demonstrates how users may use it. Do you think we should use it in a concrete example?

Reply:
Added a full minikube example.

Reply:
Addressed, thanks.